### ISSUE TITLE
log test questioned

### ISSUE THREAD

**Issue by jrpool** [cvs-health/testilo#4]
A commentator doubts that the `log` test measures anything that harms accessibility and thus recommends that its results not be used in scoring for accessibility.

**Comment by jrpool** [comment]
Testaro issue 27 (https://github.com/jrpool/testaro/issues/27) has resulted in a refinement in log reporting, with error logging messages being reported separately from logging messages as a whole. This makes it possible to count only error logging messages in a score proc. Error messages usually report that the browser was unable, or refused, to do something requested by the server. While there is no guarantee that such a failure will harm accessibility, there is also no guarantee that it will not. Protecting accessibility from unknown and unanticipated interference arguably requires updating deprecated, vulnerable, and prohibited conditions. Users who accept this argument can take logging messages into account in computing scores.
